ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -3065,6 +3108,7 @@ static int zend_jit_prologue(dasm_State **Dst)
 static int zend_jit_label(dasm_State **Dst, unsigned int label)
 {
 	|=>label:
+	|	ENDBR
```

Comment:
We don't need this for each label, only for some that are followed by ``zend_jit_prologue()``

Reply:
I add `endbr` in `zend_jit_label` to avoid "segment fault" (it's actually control protection in kernel) during debug. However, after move `endbr` from `zend_jit_align_func()` to` zend_jit_prologue()` as you suggested, the `endbr` in `zend_jit_label` looks useless because no error shown.
In following patchset rev, let's tentatively remove this.